mod-audit-server/src/main/java/org/folio/rest/impl/AuditDataAcquisitionImpl.java

```diff
@@ -0,0 +1,66 @@
+package org.folio.rest.impl;
+
+import io.vertx.core.*;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.message.ParameterizedMessage;
+import org.folio.rest.jaxrs.resource.AuditDataAcquisition;
+import org.folio.rest.tools.utils.TenantTool;
+import org.folio.services.acquisition.OrderAuditEventsService;
+import org.folio.spring.SpringContextUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.core.Response;
+import java.util.Map;
+
+import static java.lang.String.format;
+
+public class AuditDataAcquisitionImpl implements AuditDataAcquisition {
+
+  private static final Logger LOGGER = LogManager.getLogger();
+
+  private static final String TYPE = "text/plain";
```

Comment:
We need to return response errors in json format as well, you need to add some dto for errors, can use any acquisistion module as example

Reply:
fixed